spec/dor/services/client/object_spec.rb

```diff
@@ -150,6 +162,11 @@
         expect { request }.to raise_error(Dor::Services::Client::UnexpectedResponse,
                                           "conflict: 409 (#{Dor::Services::Client::ResponseErrorFormatter::DEFAULT_BODY})")
       end
+
+      it 'raises an error' do
+        expect { no_wf_request }.to raise_error(Dor::Services::Client::UnexpectedResponse,
```

Comment:
This endpoint does not return 409: see https://github.com/sul-dlss/dor-services-app/blob/master/openapi.json#L360-L373